### DISCUSSION TITLE
Adding sound support

### DISCUSSION THREAD

**Discussion by sebaxakerhtc** [ipxe/ipxe#348] [Ideas]
May be it's a good idea to add support to play sounds. It very helpful for the businesses that employ people with disabilities

**Comment by mcb30** [comment]
There are no BIOS or UEFI APIs for playing sounds, so this would be a substantial amount of work and require writing drivers for a variety of sound hardware.  It could be done, if someone has a spare $100k or so to sponsor the development.

**Reply by sebaxakerhtc** [reply]
> spare $100k

Wow! So it was a good idea, but just to this moment 🤣
Stay using Clover and OpenCore for sounds :)

**Comment by NiKiZe** [comment]
You can have bell with ANSI \b escape codes, if parsed by firmware

**Reply by sebaxakerhtc** [reply]
I don't understand, how it can help for people, who can't see at all. 
But for colors in console i use ipxe [integrated solution](https://ipxe.org/cmd/colour)
Like this:
```
  colour --basic 1 7
  cpair --foreground 7 0
```
![Снимок экрана 2021-04-30 в 13 13 51](https://user-images.githubusercontent.com/32651506/116682209-a0b09080-a9b6-11eb-9347-eb45a8df0b7a.png)
